### PR TITLE
Header & Footer Simona

### PR DESCRIPTION
**Pull request by FrontendSi**
Hello Kriste

### CODE REVIEW

File: restaurant-menu.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <body>
     <div class="main">
       <!-- Header -->
-      <header>
+      <header class="sticky">
         <div class="header-left-side"><h1>The</h1></div>
         <div class="header-right-side"><h1>MENU</h1></div>
       </header>
```
Comment (KristePe) [inline restaurant-menu.html]: Make a Navbar instead of "menu-section-row"

**Review by KristePe — COMMENTED** [review]
To make Navbar changes

**Comment by FrontendSi** [comment]
Thanks, i will update it :)

File: restaurant-menu.html (hunk 1 above)
Comment (KristePe) [reply restaurant-menu.html]: Navbar hover has to be done :)

**Pull request merged**